.github/workflows/android-debug-artifact-ondemand.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Build with Gradle
         run: |
           echo Run on-demand by ${GITHUB_ACTOR}
-          bash ./gradlew assembleDebug --stacktrace
+          bash ./gradlew testPlayDebugUnitTest assembleDebug --stacktrace
```

Comment:
Please remove `testPlayDebugUnitTest` from here as apk generation doesn't necessarily require test cases to pass.